.gitlab-ci.yml

```diff
@@ -0,0 +1,56 @@
+#*************************************************************************
+# Base class for fields
+#
+#*************************************************************************
+# Copyright 2010
+#
+# This file is part of BOUT++.
+#
+# BOUT++ is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# BOUT++ is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with BOUT++.  If not, see <http://www.gnu.org/licenses/>.
+#
+#*************************************************************************/
+
+# We define the following GitLab pipeline variables:
+#
+# GIT_SUBMODULE_STRATEGY:
+# Tells Gitlab to recursively update the submodules when cloning umpire
```

Comment:
is "umpire" a typo?

Reply:
Thanks, a short search for Umpire didn't give any meaningful results :+1:

Reply:
It's clearly an oversight: the CI setup is inspired by Umpire one... Will fix.